tests/rkt_api_service_test.go

```diff
@@ -406,3 +423,136 @@ func TestAPIServiceListInspectImages(t *testing.T) {
 		checkImageDetails(t, ctx, m)
 	}
 }
+
+func TestAPIServiceCroup(t *testing.T) {
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	svc := startAPIService(t, ctx)
+	defer stopAPIService(t, svc)
+
+	c, conn := newAPIClientOrFail(t, "localhost:15441")
+	defer conn.Close()
+
+	patches := []string{
+		fmt.Sprintf("--exec=/inspect --serve-http=localhost:%d --serve-http-timeout=%d", testServerPort, testServerTimeout),
```

Comment:
Do you use a http server only for synchronization purpose? To stop the app at the correct time?

If so, using `--interactive` and child.SendLine() like in [TestInteractive](https://github.com/coreos/rkt/blob/master/tests/rkt_interactive_test.go#L96) might be easier.